xla/service/cpu/onednn_ops_rewriter.cc

```diff
@@ -596,6 +596,9 @@ class OneDnnOpsRewriterVisitor : public DfsHloRewriteVisitor {
   }
 
   absl::Status HandleDivide(HloInstruction* divide_instr) override {
+    // TODO(intel-tf): remove this restriction after adding oneDNN softmax
+    // support in thunk runtime.
+    return absl::OkStatus();
     if (divide_instr->HasControlDependencies()) return absl::OkStatus();
```

Comment:
Please wrap the rest of the code in this function with 
```
// NOLINTBEGIN(clang-diagnostic-unreachable-code)
...
// NOLINTEND(clang-diagnostic-unreachable-code)
```
so the compiler won't complain about unreachable code.

Reply:
Done (https://github.com/openxla/xla/pull/31707/commits/00ffbac879ca18ec6c411b94bfab4c3025168dec).